src/main/java/fi/metatavu/metaform/server/rest/RealmsApiImpl.java

```diff
@@ -302,9 +368,21 @@ public Response createMetaform(String realmId, Metaform payload) throws Exceptio
       allowAnonymous  = false;
     }
     
+    Response validationResponse = validateMetaform(payload);
+    if (validationResponse != null) {
+      return validationResponse;
+    }
+
+    fi.metatavu.metaform.server.persistence.model.ExportTheme exportTheme = null;
+    if (payload.getExportThemeId() != null) {
+      exportTheme = exportThemeController.findExportTheme(payload.getExportThemeId());
+      if (exportTheme == null) {
+        return createBadRequest("Invalid exportThemeId");
+      }
+    }
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://sonarcloud.io/organizations/metatavu-oy/rules#rule_key=squid%3AS1135))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fi.metatavu.metaform-api%3Ametaform-api&pullRequest=49&issues=AWWBxMGxXKENQn1S5jLV&open=AWWBxMGxXKENQn1S5jLV)

[ISSUE_KEY]: AWWBxMGxXKENQn1S5jLV